yoda/utils/qa_processing.py

```diff
@@ -1,9 +1,16 @@
 # qa_processing.py
-import logging
+import os
+import sys
 
-from .data_reader import read_txt_data, format_text
-from utils.sambanova_endpoint import SambaNovaEndpoint
+current_dir = os.path.dirname(os.path.abspath(__file__))
+kit_dir = os.path.abspath(os.path.join(current_dir, ".."))
+repo_dir = os.path.abspath(os.path.join(kit_dir, ".."))
+
+sys.path.append(os.path.join(repo_dir,"utils"))
 
+import logging
```

Comment:
Just curious why this was moved below from the other imports.  due to the sys.path append?

Reply:
yes, only the next imports need the sys append, fixed in the next commit